packages/node_modules/@node-red/editor-client/src/js/ui/contextMenu.js

```diff
@@ -152,8 +152,16 @@ RED.contextMenu = (function() {
             }
 
         }
+
+        var direction = "right";
+        var MENU_WIDTH = 600; // can not use menu width here
```

Comment:
```suggestion
        var MENU_WIDTH = 500;
```

The regular menu width is 230px therefore i propose 500 (230 + 230 + 40px leeway)

Reply:
Changed it to 500.